ci/appveyor.yml

```diff
@@ -95,7 +95,9 @@ build_script:
 
     echo "------------------------------------------------------------------"
 
-    pip --disable-pip-version-check install $DT_WHEEL pytest docutils pandas
+    python -m pip install –upgrade pip

```

Comment:
```python -m pip install pip --upgrade```

Reply:
What is the difference with https://pip.pypa.io/en/stable/installing/#upgrading-pip ?

Reply:
hm, somehow `--` got converted to `-`, thanks will fix